src/Model/Lotto.js

```diff
@@ -0,0 +1,46 @@
+import { ERROR_MESSAGES } from "../constants/message.js";
+import { LOTTO } from "../constants/rules.js";
+
+/** 로또 당첨 번호를 관리하는 Model입니다. */
+class Lotto {
+  #numbers;
+
+  /**
+   * 로또 당첨 번호를 받아 유효성 검사를 거쳐 필드에 저장합니다.
+   * @param {string[]} numbers
+   */
+  constructor(numbers) {
+    this.#validate(numbers);
+    for (let number of numbers) {
```

Comment:
```suggestion
    for (const number of numbers) {
```
여기도 `let` 키워드를 사용하신 이유가 있으신가요??